{"name": "test_logout", "status": "broken", "statusDetails": {"message": "NameError: name 'locators' is not defined", "trace": "self = <login_test.TestLogin object at 0x041100A0>\n\n    def test_logout(self):\n    \n        try:\n            driver = self.driver\n            homepage = HomePage(driver)\n            # homepage.welcome_link_is_displayed()\n>           homepage.click_welcome()\n\ntests\\login_test.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.HomePage.flows.HomePage object at 0x043EAA60>\n\n    def click_welcome(self):\n>       self.driver.find_element(*locators.Page.welcome_link_id).click()\nE       NameError: name 'locators' is not defined\n\npages\\HomePage\\flows.py:28: NameError"}, "attachments": [{"name": "test_logout_05-07-2020_14-59-59", "source": "8551a2df-20f9-4ffd-b799-ac5940804093-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "da614ac0-54c0-4996-990a-a9e2093a12e3-attachment.txt", "type": "text/plain"}], "start": 1593950399354, "stop": 1593950400913, "uuid": "5b4c941f-a22d-4cfb-bcdd-2f1e5197bf59", "historyId": "606d9f046945fff19ea92d7b1a10f16b", "testCaseId": "e89a5a0e9aefe451ee262eba82370b01", "fullName": "tests.login_test.TestLogin#test_logout", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('test_setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-BFIIM5B"}, {"name": "thread", "value": "17100-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.login_test"}]}